hyperapp.d.ts

```diff
@@ -0,0 +1,68 @@
+export as namespace hyperapp
+
+/** @namespace [Virtual DOM] */
+
+/** The Virtual DOM representation of an Element
```

Comment:
In all the examples I've seen (and on the [JSDoc website itself](http://usejsdoc.org/about-getting-started.html)) it appears the first line of a multi-line doc comment goes on the line after `/**`. It appears [JavaDoc](https://en.wikipedia.org/wiki/Javadoc) (which I assume JSDoc is inspired from) follows the same styling convention.

Not a big deal, but just thought I'd point it out.